uberpoet/genproj.py

```diff
@@ -56,6 +56,11 @@ def make_args(args):
             '--use_wmo',
             default=False,
             help='Whether or not to use whole module optimization when building swift modules.')
+        parser.add_argument(
+            '-udl',
+            '--use_dynamic_linking',
+            default=False,
+            help='Whether or not to generate a project in which the modules are dynamically linked.  By default all projects use static linking.')
```

Comment:
Could you also note that its cocoapods only in the help.

Reply:
done!